docs/en/observability/threshold-alert.asciidoc

```diff
@@ -0,0 +1,152 @@
+[[custom-threshold-alert]]
+= Create a custom threshold rule
+
+preview::[]
+
+Create a custom threshold rule to trigger an alert when an {observability} data type reaches or exceeds a given value.
+
+. To access this page, go to **{observability}** -> **Alerts**.
+. Click **Manage Rules** -> **Create rule**.
+. Under **Select rule type**, select **Custom threshold**.
+
+[role="screenshot"]
+image::images/custom-threshold-rule.png[Custom threshold alert configuration,75%]
+
+[discrete]
+[[custom-threshold-scope]]
+== Define rule data
+
+Specify the following settings to define the data the rule applies to:
+
+* *Select a data view:* Select a data view that points to the indices or data streams that you're creating a rule for. Create a new data view by clicking *Create a data view*. Refer to {kibana-ref}/data-views.html[Create a data view] for more on creating data views.
+* *Define query filter (optional):* Use a query filter to narrow down the data that the rule applies to. For example, set a query filter to a specific host name--`host.name:host-1`--to only apply the rule to that host.
```

Comment:
Is this intentional?
![image](https://github.com/elastic/observability-docs/assets/12370520/6a01af19-2c44-417a-bda8-28c2e137464b)